spec/models/application_record_spec.rb

```diff
@@ -184,6 +184,81 @@
     end
   end
 
+  describe 'save_for_later' do
+    let :record do
+      create :application_record,
+             saved_for_later: false
+    end
+    it 'updates the saved for later attribute to true' do
+      record.save_for_later
+      expect(record.saved_for_later).to be_truthy
+    end
+    context 'mail to applicant desired' do
+      let(:mail) do
+        ActionMailer::MessageDelivery.new(JobappsMailer, :send_note_for_later)
+      end
+      it 'calls the mailer method' do
+        expect(JobappsMailer)
+          .to receive(:send_note_for_later)
+          .with(record)
+          .and_return mail
+        expect(mail).to receive(:deliver_now).and_return true
+        record.save_for_later(nil, nil, true)
+      end
+    end
+    context 'mail to applicant not desired' do
+      it 'does not call the mailer method' do
+        expect(JobappsMailer).not_to receive(:send_note_for_later)
+        record.save_for_later(nil, nil, false)
+      end
+    end
+  end
+
+  describe 'unsave' do
+    let(:record) do
```

Comment:
You alternate a fair bit in this PR between

```ruby
let :record do
```

and 

```ruby
let(:record) do
```

It'd be good to be consistent.